### PR TITLE
feat(NTD): display of all the tasks

### PR DESCRIPTION
**Pull request by mohammed-sharraf-sazim**
### Description of Change:

- Added the functionality to display all the tasks in /todo link.

### Associated Ticket Link(s):

- https://trello.com/c/CTS79DUe/3-creation-of-a-taskui

### CODE REVIEW

File: modules/todo/components/TodoForm.tsx [hunk 1]
```diff
@@ -1,53 +1,108 @@
+// import React, { useState } from "react";
```
Comment (mahir-mohsin-sazim) [inline modules/todo/components/TodoForm.tsx]: remove comments

File: modules/todo/components/TodoForm.tsx [hunk 2]
```diff
@@ -1,53 +1,108 @@
...
+//         </Button>
+//       </FormItem>
+//     </form>
+//   );
+// };
+
+// export default TodoForm;
 import React, { useState } from "react";
-import { FormItem } from "../../../shared/components/ui/form";
+import { FormItem, Form, FormField, FormLabel, FormMessage } from "../../../shared/components/ui/form";
```
Comment (mahir-mohsin-sazim) [inline modules/todo/components/TodoForm.tsx:2]: use absolute imports

**Review by mahir-mohsin-sazim — CHANGES_REQUESTED** [review]
Please review your PR first

File: modules/todo/components/TodoForm.tsx (hunk 2 above)
Comment (mohammed-sharraf-sazim) [reply modules/todo/components/TodoForm.tsx:2]: Resolved in PR#22.

**Comment by nur-zaman-sazim** [comment]
> The above feedbacks are resolved in the PR#22.

Why is this fixed in another PR but not in this one ? 
you should fix thing in older (this) PR and then just rebase the other PR(s) if you make branches of this PR

**Comment by mohammed-sharraf-sazim** [comment]
> > The above feedbacks are resolved in the PR#22.
> 
> Why is this fixed in another PR but not in this one ? you should fix thing in older (this) PR and then just rebase the other PR(s) if you make branches of this PR

SInce it was a minor change I did it in the following PR.

**Comment by nur-zaman-sazim** [comment]
> > > The above feedbacks are resolved in the PR#22.
> > 
> > 
> > Why is this fixed in another PR but not in this one ? you should fix thing in older (this) PR and then just rebase the other PR(s) if you make branches of this PR
> 
> SInce it was a minor change I did it in the following PR.

major or minor we are getting familiar with the conventions so please let's try to follow that. Don't make PR(s) dependent on each other. 
Also linking PR is done by #PR_NUMBER like, #22

**Comment by mohammed-sharraf-sazim** [comment]
> > > > The above feedbacks are resolved in the PR#22.
> > > 
> > > 
> > > Why is this fixed in another PR but not in this one ? you should fix thing in older (this) PR and then just rebase the other PR(s) if you make branches of this PR
> > 
> > 
> > SInce it was a minor change I did it in the following PR.
> 
> major or minor we are getting familiar with the conventions so please let's try to follow that. Don't make PR(s) dependent on each other. Also linking PR is done by #PR_NUMBER like, #22

Noted. I will keep that in mind.

**Review by mahir-mohsin-sazim — APPROVED** [review]

**Pull request merged**